sos/report/plugins/alternatives.py

```diff
@@ -37,13 +32,55 @@ def setup(self):
             'xinputrc'
         ]
 
-        res = self.collect_cmd_output('alternatives --list')
+        res = self.collect_cmd_output(self.alternatives_list)
         if res['status'] == 0:
             for line in res['output'].splitlines():
                 alt = line.split()[0]
                 if alt not in ignore:
                     alts.append(alt)
-            disp_cmd = "alternatives --display %s"
+            disp_cmd = "%s --display %s" % (self.alternatives_cmd, "%s")
             self.add_cmd_output([disp_cmd % alt for alt in alts])
 
+
+class RedHatAlternatives(Alternatives, RedHatPlugin):
+
+    packages = ('chkconfig',)
```

Comment:
I think this comes from before plugins supported `commands` for enablement, and what's more is `chkconfig` has been obsolete for a long time at this point.

In any case, we can either drop this, or change this to `packages = ('alternatives', )`